### PR TITLE
Nightly Tests - Improvements

### PR DESCRIPTION
**Pull request by revitteth**
- updated run order and timing
- use different L2 account for bench tests

### CODE REVIEW

File: .github/workflows/nightly-bridge-tests.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ name: Nightly - Bridge Tests
 
 on:
   schedule:
-    - cron: '0 0 * * *'  # run at midnight
+    - cron: '0 0 * * *'  # run at midnight (take around 2m to pass)
```
Comment (mandrigin) [inline .github/workflows/nightly-bridge-tests.yml]: That is a useless test we need to remove it

File: .github/workflows/nightly-eth-bench.yml [hunk 2]
```diff
@@ -2,7 +2,7 @@ name: Nightly - ETH Bench
 
 on:
   schedule:
-    - cron: '0 1 * * *' # run at 1am to stagger tests
+    - cron: '0 1 * * *' # run at 1am to stagger tests (take around 15m to pass)
```
Comment (mandrigin) [inline .github/workflows/nightly-eth-bench.yml]: 1:30

File: .github/workflows/nightly-bridge-erc20.yml [hunk 3]
```diff
@@ -2,7 +2,7 @@ name: Nightly - Bridge ERC20 L2->L1
 
 on:
   schedule:
-    - cron: '0 1 * * *' # run at 1am to stagger tests
+    - cron: '30 0 * * *'  # run at 12:30am
```
Comment (mandrigin) [inline .github/workflows/nightly-bridge-erc20.yml]: 12:45

File: .github/workflows/nightly-eth-bench.yml (hunk 2 above)
Comment (revitteth) [reply .github/workflows/nightly-eth-bench.yml]: I guess the point was to avoid this running at the same time as ansible tests?

**Review by mandrigin — APPROVED** [review]

**Pull request merged**